scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd

```diff
@@ -13,13 +13,14 @@ SPDX-License-Identifier: Apache-2.0
         <xs:documentation xml:lang="en">
             Version 1.0 Release 2021/06/16
             Version 1.1 Release 2022/01/21
+            Version 2.1 Release 2022/06/08
         </xs:documentation>
     </xs:annotation>
 
     <!--
     SCL Name element.
     -->
-    <xs:simpleType name="tCompasSclName">
+    <xs:simpleType name="tSclName">
```

Comment:
Why is this name Changed? This is now back again to the original name.

Reply:
@jeanetiennelemaire @DamienPernet I let you answer

Reply:
Mistake have been corrected in the new version of the xsd file